estuary_updater/handlers/koji.py

```diff
@@ -38,17 +42,21 @@ def handle(self, msg):
         """
         topic = msg['topic']
 
-        if topic == '/topic/VirtualTopic.eng.brew.build.complete':
-            self.build_complete_handler(msg)
+        if topic == '/topic/VirtualTopic.eng.brew.build.complete' or \
```

Comment:
Optional: Storing these topics in a list/tuple/set and then doing `if topic in build_status_topics` would be cleaner.